### DISCUSSION TITLE
Superposing plots

### DISCUSSION THREAD

**Discussion by Jesusmunozmontecinos** [mtex-toolbox/mtex#715] [Ask Anything]
Dear all,

I come again to you in order to ask about superposing plots.
My idea is to plot a mis2mean map of Lawsonite (or whatever mineral, attached picture) over the band contrast image  (in gray scale or black and white) without loosing the color map and colorbar (and also the grain boundaries) of my mis2mean map. I have not managed to do that with the documentation found here https://mtex-toolbox.github.io/EBSDPlotting.html

Code: 
region = [15 13 2323 2213];
condition = inpolygon(ebsd,region);
ebsd = ebsd(condition)
[grains,ebsd('indexed').grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'));
[grains,ebsd('indexed').grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'),'angle',10*degree);
F = halfQuadraticFilter;
ebsdS = smooth(ebsd('indexed'),F,'fill',grains);
[grains,ebsdS('indexed').grainId,ebsdS.mis2mean] = calcGrains(ebsdS('indexed'));
------- **all of this is what I would have to superpose with a black and white or grayscale band contrast image as background, 
            mainly for aesthetic reasons**
plot(ebsdS('Lawsonite'),ebsdS('Lawsonite').mis2mean.angle./degree)
mtexColorbar
hold on
plot(grains.boundary,'edgecolor','k','linewidth',1.5)
hold off
CLim(gcm,[0 10])
mtexColorMap jet

% the result of this has been attached 
![image](https://user-images.githubusercontent.com/76631957/105089288-35911d00-5a9d-11eb-94e4-91dfc1b19625.png)

-------

Can someone help me with my little code, where should I modify in order to add the band contrast image background?

Hopping you a nice week,

Jesús

**Comment by kilir** [comment]
Hi Jesus,
do you mean something like that https://groups.google.com/g/mtexmail/c/JY-hbvqXYo8/m/cRcGR0AyCAAJ ?
Cheers, Rüdiger
